src/views/Team/Outlet.vue

```diff
@@ -24,6 +24,9 @@
                     <ion-icon size="small" :icon="editIcon" />
```

Comment:
I think the changes in this file don't belong into this PR.

Reply:
Resolved